library/system/src/modules/Kernel.rb

```diff
@@ -248,6 +253,10 @@ def ParseInstallationKernelCmdline
       @cmdline_parsed = true
       return if !(Stage.initial || Stage.cont)
       tmp = Convert.to_string(SCR.Read(path(".etc.install_inf.Cmdline")))
+      # live installation does not create /etc/install.inf (bsc#793065)
+      if Mode.live_installation
+        tmp = Convert.to_string(SCR.Read(path(".target.string"), "/proc/cmdline"))
```

Comment:
I know about the agent. The reason not to use it is to process cmdline from install.inf the same as from /proc/cmdline during live install - to get a single string in both cases, parsing them via the same parser. Otherwise, one would be parsed via agent and another one via ruby code.
If-else - agreed, will adjust.